daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/EngineConfig.scala

```diff
@@ -7,24 +7,64 @@ package engine
 import java.nio.file.Path
 
 import com.daml.lf.language.{LanguageVersion => LV}
-import com.daml.lf.transaction.{TransactionVersions, TransactionVersion => TV}
+import com.daml.lf.transaction.{TransactionVersions, TransactionVersion}
+import com.daml.lf.value.ValueVersion
 
+/**
+  * Engine configuration. This describe the  version of language and
+  * transaction the engine is allowed to read and write. This also
+  * parameterized debugging option.
```

Comment:
I'm not sure I understand what you want to say here. Is this particular class parametrized by a debugging option or are the debugging options parametrized by this class?